ExilenceNextApp/src/utils/snapshot.utils.ts

```diff
@@ -51,10 +51,8 @@ export const mapSnapshotsToStashTabPricedItems = (snapshot: Snapshot, stashTabs:
     });
 };
 
-export const getSnapshotCardValue = (snapshotCount: number) => {
-  let label = `${snapshotCount}${snapshotCount >= 1000 ? '+' : ''}`;
-  return label;
-};
+export const getSnapshotCardValue = (snapshotCount: number) =>
+  `${snapshotCount}${snapshotCount >= 1000 ? '+' : ''}`;
```

Comment:
Dont we need a return here or am I mistaken?

Reply:
Technically `() => { return 'foo' }` is the same as `() => 'foo'`.

Reply:
Gotcha, thanks for clearing it up!